### DISCUSSION TITLE
Webhooks not always processed

### DISCUSSION THREAD

**Discussion by ya-ku** [postalserver/postal#1957] [Help with using Postal]
Hey everyone,

I'm pretty new in using Postal and stumbled across a problem regarding webhooks. I set up two webhooks (MessageBounced & MessageDeliveryFailed), however Postal only partially triggers these webhooks. During this week I sent >20k mails and from a total of 3.5k bounced and hard failed mails only 2.3k got processed via the webhooks. I realized that the remaining 1.2k webhook requests are queued in the belonging table inside the database and not being called.

#265 & #330 are appointing the same issue that I'm having right now. They proposed a temporary fix for the problem but since then version 2 of Postal got released and maybe there is a better solution for this issue. Anyone experienced the same problem and has a good way to solve this?

**Comment by willpower232** [comment]
If those issues give you a resolution, I don't think there will be a better solution.

**Reply by ya-ku** [reply]
You're probably right, nevertheless wanted to check that after all this time I don't miss out on something. Thanks a lot. 

But I have a short follow up question regarding webhooks. I would like to limit the outgoing webhooks because sometimes the receiving server gets overloaded by the sheer size of requests. Do you know by any chance which line to change in order to send the webhooks in batches (limit(50) for example)?

**Reply by willpower232** [reply]
I don't think that would be possible because Postal doesn't have any concept of batches of webhooks so it would be a bit more than one line of change.
